Adyen/Assets/ar.lproj/Localizable.strings

```diff
@@ -67,6 +67,7 @@
 "adyen.voucher.finish" = "Finish";
 "adyen.pix.instructions" = "افتح التطبيق باستخدام مفتاح PIX المسجل، واختر الدفع باستخدام PIX وامسح رمز الاستجابة السريعة ضوئيًا أو انسخ الرمز والصقه";
 "adyen.pix.expirationLabel" = "لديك %@ للدفع";
+"adyen.pix.copyButton" = "";
```

Comment:
Better skip empty translations - it will fallback to English

Reply:
is there a way to do this automatically?

Reply:
you can check if the string is empty before adding in the StringGenerator script

Reply:
what if it is empty for some languages and not others?

Reply:
It will fallback to English, right?

Reply:
yes. if the result string is empty, or was missing from the table - it will fallback to en-US

Reply:
so I guess it is better to not include empty keys?

Reply:
I still don't understand how that would be done. Manually?